djangocms_googlemap/models.py

```diff
@@ -245,6 +245,8 @@ def get_short_description(self):
             display += '{0}, '.format(self.address)
         if self.lat and self.lng:
             display += '{0} / {1} '.format(self.lat, self.lng)
+        # remove trailing comma if lat or lng are not provided
+        display = display.rstrip(', ')
```

Comment:
@czpython then you'd have the problem with the first ,:
![image](https://cloud.githubusercontent.com/assets/270307/19974011/54bd18dc-a1e7-11e6-9be6-5cc6b7569fe5.png)

would have solved it that way if its so easy :D